### ISSUE TITLE
loss layers when training?

### ISSUE THREAD

**Issue by CongWeilin** [kpzhang93/MTCNN_face_detection_alignment#12]
I want to train the data by myself. 
My method: 
input data: four kinds of training data( includes positive, negative, part, landmarks)
for different data, only backward specific loss layers.(eg. For positive update regression Loss and landmark loss, for negative update classification loss , for landmark face update landmark loss)
However, in the paper, `We use det:box:landmark = 1:0.5:0.5 in P-Net and R-Net`, how can i implement it?
1. change loss layer `Weigh_Loss` to 2:1:1?
2. make training data pos:neg:part:land = 3:2:1:1?

I wish to know the two method above is it correct? 
What is your method?
What the data proportion（pos:neg:part:land = ?:?:?:?）

Thanks a lot

**Comment by xingtel** [comment]
如何训练，实际 这个 思路 可以用到 车牌 识别上，很有价值呀
